hack/jenkins/job-configs/global.yaml

```diff
@@ -29,6 +29,33 @@
                     else
                       curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-to-gcs.sh" | bash -
                     fi
+                - shell: |
+                    curl -fsS --retry 3 "https://raw.githubusercontent.com/kubernetes/kubernetes/master/hack/jenkins/upload-finished.sh" > upload-finished.sh
+                    chmod 777 upload-finished.sh
+                - conditional-step:
+                    condition-kind: current-status
+                    condition-worst: SUCCESS
+                    condition-best: SUCCESS
+                    steps:
+                        - shell: './upload-finished.sh SUCCESS'
+                - conditional-step:
+                    condition-kind: current-status
+                    condition-worst: UNSTABLE
+                    condition-best: UNSTABLE
+                    steps:
+                        - shell: './upload-finished.sh UNSTABLE'
+                - conditional-step:
+                    condition-kind: current-status
+                    condition-worst: FAILURE
+                    condition-best: FAILURE
+                    steps:
+                        - shell: './upload-finished.sh FAILURE'
+                - conditional-step:
+                    condition-kind: current-status
+                    condition-worst: ABORTED
+                    condition-best: ABORTED
```

Comment:
I guess to answer my own question: [probably](http://javadoc.jenkins-ci.org/hudson/model/Result.html). Not sure if we should bother with `NOT_BUILT` too?